### ISSUE TITLE
Geberate keys for objects

### ISSUE THREAD

**Issue by sdanko** [feliixx/mgodatagen#11]
is it possible to use generators for object keys?

**Comment by feliixx** [comment]
No, currently it's not possible to use generators as object keys. However, it should be quite simple to implement this. I'll try to take a look at this next week !
